impeller/display_list/display_list_dispatcher.cc

```diff
@@ -448,6 +448,10 @@ void DisplayListDispatcher::setColorFilter(
         return FilterContents::MakeLinearToSrgbFilter({input});
       };
       return;
+    case flutter::DlColorFilterType::kComposedFilter:
```

Comment:
Also, check the dl_testing.h to make sure that it can print out the new filter type.